### ISSUE TITLE
In pre-training a model how can I resume from last saved model?

### ISSUE THREAD

**Issue by puraminy** [huggingface/transformers#13965]
Using flax example for training a T5 model from scratch, I am training a model using:

```
thon run_t5_mlm_flax.py \
	--output_dir="/content/drive/MyDrive/Pouramini/" \
	--cache_dir="/content/drive/MyDrive/cache/" \
	--preprocessing_num_workers="20" \
  --model_type="t5" \
	--config_name="/content/drive/MyDrive/Pouramini/" \
	--tokenizer_name="/content/drive/MyDrive/Pouramini/" \
	--dataset_name="oscar" \
	--dataset_config_name="unshuffled_deduplicated_fa" \
	--max_seq_length="256" \
	--per_device_train_batch_size="32" \
	--per_device_eval_batch_size="32" \
	--adafactor \
	--learning_rate="0.005" \
	--weight_decay="0.001" \
	--warmup_steps="2000" \
	--logging_steps="500" \
	--save_steps="5000" \
	--eval_steps="50000" \
  --resume_from_checkpoint=True
```

Sometimes, the training interrupts and I must run the command again. But while I have specified `resume_from_checkpoint` it seems it trains from scratch, because the loss it shows is higher than the loss of the last saved model. What did I miss in the code above?
I thought to add:

```
  --model_name_or_path="/content/drive/MyDrive/Pouramini/" \
```
Is it necessary?

**Comment by LysandreJik** [comment]
cc @sgugger

**Comment by sgugger** [comment]
It's Flax, so it's more up to @patil-suraj aisle than mine :-)
